static/js/collapsible-filters/interactions.js

```diff
@@ -123,6 +123,18 @@ export default class Interactions {
       eventType: 'update',
       storageKey: 'query',
       callback: () => {
+        const repeatedSearchTriggers = [
+          'filter-component',
+          'pagination',
+          'suggest',
+          'initialize',
+          'query-parameter'
+        ];
+        const queryTrigger = ANSWERS.core.storage.get('queryTrigger');
```

Comment:
Can we add queryTrigger to the new storage-keys.js and reference from there?

Reply:
yes definitely, I kinda want to import these directly from the SDK src in the future (or maybe provide some export in the SDK) but that's a good middle ground!

Reply:
oh wait I thought you meant the actual query triggers themselves, will also update the storage key